README.md

```diff
@@ -1,393 +1,296 @@
 
-### Launching browsers
+#### `timeout`
+Type: `Number`  
+Default value: `120`  
+The time (in seconds) to wait for a test page to report back before an error is thrown. This timeout might, for example, occurr when the given file doesn't exist, the browser didn't start, or the localhost tunnel wasn't running.
 
-The `launch` and `browsers` options are used to set the environment and the browsers you want
-to start. [Launchpad](https://github.com/ekryski/launchpad) is the browser launcher library used
-by Testee which allows you to start most locally installed browsers as well as Browserstack workers
-and even browsers on remote systems running the [Launchpad Server](https://github.com/ekryski/launchpad#the-launchpad-server).
 
-The most common case will be launching a local browser (which is also the default) with no settings:
+### Browser Settings
 
-```js
-{
-  "browsers" : [ "firefox", "safari" ]
-}
-```
+**Note:** Depending on your OS, the target browsers should not already be open/running before using this library. Electron and PhantomJS are the exceptions, as they can always be started multiple times.
 
-#### Browserstack
+#### `browsers`
+Type: `Array`  
+Default value: `['phantom']`  
+The browsers that will be used to run tests. For local browsers, use a browser name string (see [launchpad](https://npmjs.com/launchpad) for more info). For a remote/BrowserStack browser, use a [browser object](https://npmjs.com/browserstack#browser-objects). Mixing both types is not currently possible.
 
-Browserstack hosts virtual machines running specific versions of web browsers. It is an extremely useful
-tool for cross-browser testing running a remote desktop in you browser. To use BrowserStack via the configuration
-API you need to provide a username and password:
+#### `launch`
+Type: `Object`  
 
-```js
-{
-  "launch": {
-    "type": "browserstack",
-    "username": "your browserstack username",
-    "password": "your browserstack password",
-    "version": "browserstack API version (recommended: 4)"
-  }
-}
-```
+#### `launch.password`
+Type: `String`  
+Default value: `undefined`  
+Your BrowserStack API key.
 
-To start a worker, you must provide a valid [browser object](https://github.com/scottgonzalez/node-browserstack#browser-objects) to the `browser` option:
+#### `launch.type`
+Type: `String`  
+Default value: `'local'`  
+The test environment. Possible values are `'local'` and `'browserstack'`.
 
-```js
-{
-  browsers: [{
-    os: "win",
-    os_version: "7.0",
-    browser: "ie",
-    browser_version: "8.0"
-  }, {
-    os: "win",
-    os_version: "7.0",
-    browser: "ie",
-    browser_version: "11.0"
- }]
-}
-```
+#### `launch.username`
+Type: `String`  
+Default value: `undefined`  
+Your BrowserStack username.
 
-An example configuration that runs your tests on an iPad Mini and Samsung Galaxy S3 emulator using BrowserStack
-in a CI environment (outputting XUnit logs) could look like this:
```

Comment:
I saw where the example is, but it doesn’t have any information about what it does, which this paragraph helpfully provided.